bin/cli.js

```diff
@@ -165,6 +168,22 @@ const generateComponentIndex = package => {
   );
 };
 
+const generateComponentThemeIndex = package => {
+  const indexFile = `${packagePath(package)}/theme/components/index.js`;
+  if (existsSync(indexFile)) {
+    unlinkSync(indexFile);
+  }
+  const fd = openSync(indexFile, "a");
+  listComponentsInPackage(package).forEach(componentName => {
```

Comment:
I think this should walk the subfolders of `${package}/theme/components/` rather than `listComponentsInPackage()`, because there may be components for which no theme subfolder exists (bc the author would have deleted it after it was generated). 

Eg, see `globalStyle`, `navBarSection`, and `textarea`. Lines for these were auto-generated for `packages/core/theme/components/index.js` in this commit, but the corresponding theme subfolders don't exist (and shouldn't, in these cases).

Reply:
Yeah, I was considering this also. Will make that change.

Reply:
Finally pushed this change, sorry for the delay.